src/sentry/static/sentry/app/components/group/externalIssueActions.jsx

```diff
@@ -234,13 +246,38 @@ class ExternalIssueActionList extends AsyncComponent {
             <Modal.Header closeButton>
               <Modal.Title>{`${selectedIntegration.provider.name} Issue`}</Modal.Title>
             </Modal.Header>
+            <ul
+              className="nav nav-tabs"
+              style={{'borderBottom': '1px solid rgb(221, 221, 221)'}}
+            >
+              <li className={action == 'create' ? 'active' : ''}>
+                <a id="create" onClick={this.handleClick}>
+                  Create
+                </a>
+              </li>
+              <li className={action == 'link' ? 'active' : ''}>
+                <a id="link" onClick={this.handleClick}>
+                  Link
+                </a>
+              </li>
+            </ul>
             <Modal.Body>
-              <ExternalIssueForm
-                group={this.props.group}
-                integration={selectedIntegration}
-                action={action}
-                onSubmitSuccess={this.closeModal}
-              />
+              {action == 'create' && (
```

Comment:
Let's use triple compare (`===`)

Reply:
I'm suprised we don't have a linting rule for this.

Reply:
I was hesitant to turn it on and mass format blindly because there could be instances where we rely on non-strict comparisons. I was thinking we should have two set of lint rules where pre-commit hooks are stricter than our CI so it'd be easier to transition to new rules.